aiida/orm/nodes/data/upf_to_json/__init__.py

```diff
@@ -0,0 +1,3 @@
+""" UPF converter """
+
+from .upf_to_json import upf_to_json
```

Comment:
The right place for these tools (we're slowly moving also other things there) is not in `aiida/orm/data` but in `aiida/tools/data/upf/...`. Could you move the logic there?

Reply:
@giovannipizzi Yes, this is the entire code we have for UPF to json conversion. Although moving it to a separate dependency might be a good idea.

Reply:
Moving the files to aiida/tools/data/upf seems to trigger a circular import if I then import the function in `orm/nodes/data/upf.py`.